app/mender_test.go

```diff
@@ -1356,31 +1359,37 @@ func TestUploadPauseStatus(t *testing.T) {
 
 	for name, test := range tests {
 		t.Run(name, func(t *testing.T) {
+			ms := store.NewMemStore()
 			pool := NewControlMap(
-				store.NewMemStore(),
-				conf.DefaultUpdateControlMapBootExpirationTimeSeconds)
+				ms,
+				conf.DefaultUpdateControlMapBootExpirationTimeSeconds,
+				conf.DefaultUpdateControlMapBootExpirationTimeSeconds,
+			)
 			pool.Insert(&updatecontrolmap.UpdateControlMap{
 				ID:       "foo",
 				Priority: 1,
 				States: map[string]updatecontrolmap.UpdateControlMapState{
 					test.state: updatecontrolmap.UpdateControlMapState{
 						Action:           "pause",
-						OnActionExecuted: "pause",
+						OnActionExecuted: "continue",
```

Comment:
Do these actually work well together? One weakness I discovered (at least in earlier revisions) is if `Action: pause` and `OnActionExecuted: continue`, then it might somewhat randomly decide to continue simply because the loop spins and the next action gets set. I concluded that it's not very important to fix because the combination of those two is quite weird and not too well defined.

But it raises the question: What is the reason for switching it here, and does it really do what you think it does?

Reply:
I decided to simply remove it completely.
The reason it was changed, was to reign in the goroutines resulting from the `go spinEventLoop`. I don't see how changing this should have any effect on the test itself.